### ISSUE TITLE
Overview should be time based

### ISSUE THREAD

**Issue by bkaless** [systopia/de.systopia.campaign#30]
The overview should have a start-date and end-date to narrow the Chart and the Values in the overview.

**Comment by bjendres** [comment]
Currently the start and end date are determined by the first and last contribution linked to the campaign.

@bkaless: What interface would you propose to change this interval?

**Comment by bkaless** [comment]
I wish two fields: from and to. If one field is empty, the first or the last contribution should be used.

**Comment by bjendres** [comment]
@bkaless: Are we talking about a dynamic slider to "play with" or a general time span that applies to a specific campaign. The latter could probably be achieved by evaluating a campaign's start and end date, but the first might be more tricky....

**Comment by Cullpepper** [comment]
I agree with bkaless - often, a report is needed that fits within a calendar or fiscal year. However, contributions may fall outside the reporting range and still belong to the campaign. (For example, It's not uncommon for sponsorships to be paid in advance, or for direct-mail appeals to return months after the campaign ended.)

Date selection boxes "from" and "to" would be most helpful for this type of scenario. 

Thanks again, this is a wonderful extension.

**Comment by bjendres** [comment]
@Cullpepper: again: where do you guys want that box: to dynamically adjust the KPIs or as a general per-campaign setting?

**Comment by bkaless** [comment]
I personaly want it at the overview-page of a campaign in the campaign-manager. It is sufficient if it is a date field instead of a slider. Do you know, what I mean?
